tests/Http/Api/ActivityApiTest.php

```diff
@@ -86,14 +78,13 @@ public function testActivityIndexWithCampaignRunIdQuery()
     {
         $signup = factory(Signup::class)->create();
 
-        $this->json('GET', $this->activityApiUrl . '?filter[campaign_run_id]=' . $signup->campaign_run_id);
+        $this->get('api/v2/activity?filter[campaign_run_id]=' . $signup->campaign_run_id);
```

Comment:
Yeah, it's a little subjective! I think this may just have been a side-effect of not being as familiar with the codebase, but I found myself consistently scrolling up in each file to remember where I was as I read the tests. My general approach with testing is to be as explicit as possible so it's really easy to follow exactly what steps are being performed when looking at a test in isolation.

Definitely up for having a discussion at retro & reverting based on team opinion!